integration_tests/helpers/walletProcess.js

```diff
@@ -8,18 +8,8 @@ const WalletClient = require('./walletClient');
 
 class WalletProcess {
 
-    constructor(name, baseNodeAddress, baseNodePort, baseNodeGrpcPort) {
-            this.name = name;
-            this.nodeFile = "cwalletid.json";
-            this.baseNodeAddress = baseNodeAddress;
-            this.baseNodePort = baseNodePort;
-            this.baseNodeGrpcPort = baseNodeGrpcPort;
-    }
-
-    async preInit() {
-        this.baseNodeAddress = "127.0.0.1";
-        this.baseNodePort = await getFreePort(19000, 25000);
```

Comment:
Surely removing this prevents creating multiple wallets?

Reply:
That function does not add any value; it only grabs ports and does nothing with it. Having that function in caused an issue with ports in uise.